3rdPartyCypress/cypress/integration/worldServiceCoUkRedirect.js

```diff
@@ -0,0 +1,39 @@
+import services from '../../../src/app/lib/config/services';
+
+Object.keys(services).forEach(index => {
+  const service = index;
+
+  const notWSServices = ['news', 'cymrufyw', 'naidheachdan', 'default']; // Not WS
```

Comment:
```suggestion
  const notWSServices = ['news', 'cymrufyw', 'naidheachdan', 'default', 'loadableConfig']; // Not WS
```

Reply:
This doesn't fail when I run it.

Reply:
Same here. That's interesting. Resolving comment.